field/composite.go

```diff
@@ -424,12 +426,20 @@ func (f *Composite) unpackSubfieldsByTag(data []byte) (int, error) {
 			tagBytes = f.spec.Tag.Pad.Unpad(tagBytes)
 		}
 		tag := string(tagBytes)
-		if _, ok := f.spec.Subfields[tag]; !ok {
+		if _, ok := f.spec.Subfields[tag]; !ok && !skipUnknownTLVTags(f) {
```

Comment:
I would move the code that changes `offset` from the block below into this `if` block. If `tag` is not defined in the spec, it will not be in the `f.subfields[tag]`. It means we can either error or skip it right here (in this block) and `continue` without calling the code below.

Reply:
Done.
Thank you for the suggestion.